oai_agents/agents/rl.py

```diff
@@ -2,7 +2,7 @@
 from oai_agents.common.arguments import get_arguments
 from oai_agents.common.networks import OAISinglePlayerFeatureExtractor
 from oai_agents.common.state_encodings import ENCODING_SCHEMES
-from oai_agents.common.tags import AgentPerformance, TeamType, TeammatesCollection
+from oai_agents.common.tags import AgentPerformance, TeamType, TC
```

Comment:
While TC is an abbreviation, TeamType and AgentPerformance are not. It will be nice to have them with the same style. 
What do you think?

Reply:
How about renaming it by TeammatesCollectionType?

Reply:
Updated it to TeammatesCollection